app/views/application/_location_form.html.erb

```diff
@@ -22,7 +24,12 @@
     } %>
   <% end %>
 
-  <form method="post" id="local-locator-form" class="location-form govuk-!-margin-bottom-9">
+  <%= content_tag(:form,
```

Comment:
Can you add something to the commit message about why this has changed to a `content_tag`? For example, as this is a form, why wasn't a `form_with` used?